src/seedu/addressbook/data/person/UniquePersonList.java

```diff
@@ -113,6 +113,14 @@ public void clear() {
         internalList.clear();
     }
 
+    /**
+     * New method to get size of address book :D
```

Comment:
Do note that your header comment should begin with a verb - e.g. Gets size of address book. This is the standard java coding convention that we are following.